lab_prog_2.cpp

```diff
@@ -5,11 +5,11 @@ int main()
     int n,isum=0,i;
     cout<<"enter the value of n ";
     cin>>n;
-    for(i=1;i<=n;i++)
+    for(i=1;i<=n;i++)//sum of 1 to n number
     {
         isum+=i;
```

Comment:
Explain how this statement works

Reply:
first, i is assigned to a value 1 then it will check for condition, here the condition as 1<=1 is true then control is entered into for loop and it will increment the value of isum by one Now the isum value is 1. then the control will increment the value of i now the i value is 2, then it will check the condition and now condition is false as 2 !<= 1, so the control will come out of loop.

Reply:
first, i is assigned to a value 1 then it will check for condition, here the condition as 1<=1 is true then control is entered into for loop and it will increment the value of isum by one Now the isum value is 1. then the control will increment the value of i now the i value is 2, then it will check the condition and now condition is false as 2 !<= 1, so the control will come out of loop.